environments/centos-9/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM rockylinux:9
```

Comment:
Since CentOS 9 doesn't actually exist, we should call this directory `el-9` to refer to the fact that support all Enterprise Linux derivatives.